neofetch

```diff
@@ -1560,6 +1560,9 @@ get_de() {
         ;;
     esac
 
+    if [ "$KDE_SESSION_VERSION" -ge "4" ]; then
```

Comment:
Instead of an if block, use `(( KDE_SESSION_VERSION > 4 )) && [...]` as it is a bash syntax.

Reply:
KDE was renamed to Plasma during the KDE 4 days. https://en.wikipedia.org/wiki/KDE_Plasma_4